data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1831 @@
+			`	When the engineers return, they put your outfit scanner back in place, and the Heliarch agents say their goodbyes, reassuring you that the Quarg shouldn't attack you for this. "If interested in doing some investigation of your own, you wish, maybe land on their rings again, you could," the Kimek suggests. "Maybe find out something of interest for us, you will."`
+			`	He wishes you safe travels, and leaves with his colleagues.`
+
+mission "Quarg Interrogation"
+	invisible
+	landing
+	source
+		government "Quarg"
+		not planet "Humanika"
+	destination "Earth"
+	to offer
+		has "Heliarch Recon 2-B: done"
+	on offer
+		conversation
+			`As you land on <origin>, a Quarg is patiently waiting right outside. Seeing that it won't go away, you head outside to speak with it.`
+			`	"Hello, <first> <last>," it says.`
+			choice
+				`	"Hi, can I help you?"`
+				`	"Is there some sort of problem?"`
+			`	It blinks, then slowly looks to a Wardragon parked farther away. "You were seen landing on some of our worlds and rings, bringing about strange scanning mechanisms, and were also found to be scanning some of our ships. Why is that?"`
+			choice
+				`	"I was just curious about your species."`
+					goto lie
+				`	"What are you talking about? I never did any of that."`
+					goto lie
+				`	"Scanning? I'm sorry, I think my ship just had some malfunctions. I didn't mean to offend."`
+					goto lie
+				`	"The Heliarchs asked me to do it."`
+					goto truth
+			label lie
+			`	Around ten seconds pass without it saying anything, simply staring at your eyes and blinking slowly. "I see." Without another word, it walks away, headed to the Wardragon.`
+				decline
+			label truth
+			`	"And why did you agree? Do you wish to join their ranks, perhaps?" It asks.`
+			choice
+				`	"I just wanted to help. It's not like I destroyed your ships or something."`
+				`	"They told me that the Quarg keep others in 'separate boxes', and that you hinder their development."`
+					goto hinder
+			`	"Perhaps not, but you agreed to collect intel on us and our worlds, which hold countless civilians, and deliver the findings to a group that hates us."`
+				goto end
+			label hinder
+			`	"And you believe that? Have we restricted your movement? Chastised you for interacting with the Hai or Korath? Even after you agreed to help those foolish, tyrannical Heliarchs, are you not here, in one of our worlds, peacefully conversing with a Quarg?"`
+			label end
+			`	It blinks a few more times as a few small stripes on its face rapidly alternate between shades of deep blue and bright red, then ends with, "It would be most wise to cut off all your ties with these 'Heliarchs', human." It walks away, headed to the Wardragon it looked at before.`
```

Comment:
Just depart and have that be that? Or depart and follow the player? I'm not sure on the latter since it'd be odd if it departed only to follow you to the Coalition and get killed.

Reply:
No he shouldn't follow the player indeed, just make sure he leaves the system without any funny business.

Reply:
I did a thing, tell me if it doesn't work right.

Reply:
Seems good, but maybe he should be staying too? 
Also I think he should have 
```to despawn 
  has "Heliarch Recon 3-A: offered"```
So that he doesn't stay around forever, staying in that system he can easily check if you're doing smth you shouldn't be, and its mostly to impress you probably that he goes in his Wardragon

Reply:
I gave the mission a deadline of 1 so that it fails pretty immediately, wouldn't that already despawn the ship?

Reply:
You're right, I just didn't notice, it will indeed despawn the ship the next time you land on a planet